### PR TITLE
Handle WSEGITER keyword data and WELSEGS keyword item 6 for Eclipse compatible restart file

### PR DESCRIPTION
**Pull request by jalvestad**
This PR handles the output to the Eclipse compatible restart file of:
 i.  WSEGITER keyword parameters and
 ii. WELSEGS keyword item 6 parameter.

These parameters are needed for an Eclipse restart from flow in order to obtain the appropriate values for the actual parameters.

The PR is ready for comments and review.

NOTE: The PR includes code, which is commented out, that is needed to ouput WELSEGS item7 values to the restart file. However, since flow currently only support the HO (homogeneous flow) option, this code is inactive at the moment and therefore may be deleted if considered inappropriate.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
For future reference: https://ci.opm-project.org/job/opm-common-PR-builder/2489/

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this opm-tests=386 please

**Review by bska — APPROVED** [review]
This looks good to me.  I think we can merge this as is.  Any other comments, @joakim-hove?

File: opm/output/eclipse/VectorItems/well.hpp [hunk 1]
```diff
@@ -58,6 +58,18 @@ namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems
                          //   Value 0 for regular wells
                          //   Value #segments for MS wells
 
+            MSW_PlossMod  = 81, // Pressure loss model for well segments
+                         //   = 0 for regular wells
```
Comment (joakim-hove) [inline opm/output/eclipse/VectorItems/well.hpp:62]: Could indicate that the 0,1,2 are *values* - in contrast to the 81 which is an *index*.

**Comment by joakim-hove** [comment]
> Any other comments, @joakim-hove?

No this looks fine. I am a little frustrated we have not found a good system for the numerous "int(ecl) &#8596; enum(opm)" conversions, but this PR is not to blame.

File: opm/output/eclipse/VectorItems/well.hpp (hunk 1 above)
Comment (jalvestad) [reply opm/output/eclipse/VectorItems/well.hpp]: Thank you for good comment, I have changed the comments, to clarify their meaning, hope it is OK now?

**Comment by joakim-hove** [comment]
This can be merged!

**Comment by bska** [comment]
> This can be merged!

Thanks!  I'll do that right away, first by bringing in the new reference data in OPM/opm-tests#386

**Comment by bska** [comment]
Merging this now per maintainer approval.  Thanks a lot!

**Pull request merged**